pkg/lib/cockpit-components-file-autocomplete.jsx

```diff
@@ -270,6 +266,9 @@ export class FileAutoComplete extends React.Component {
         else
             controlClasses += "caret";
 
+        if (this.pendingCallback)
+            classes += " pending-callback";
```

Comment:
This class isn't being used anywhere?

Reply:
it is being used in cockpit-podman, and was needed for the tests to make it fully testable, once this class is removed we're sure that the onChangeCallback was called.We don't use it in cockpit tests, but maybe we should.

Reply:
I see, thanks! So let's keep that.